cmds/core/sluinit/uinit_linux.go

```diff
@@ -1,6 +1,7 @@
-// Copyright 2019 the u-root Authors. All rights reserved
```

Comment:
this seems unrelated to cpuid?

Reply:
Yes, it is. Since it is not really important I will remove this from the PR.